app/assets/stylesheets/components/_v-nav.scss

```diff
@@ -142,6 +141,7 @@ $pane-side-padding: 42px;
         &__item, &__dropdown-item {
           @include responsive-styles('font-size', 25px, 30px, 30px);
```

Comment:
You don't need to change this, but do you prefer not leaving a line between include/extend statements and standard styling? I generally leave a line, but am happy to do this if that's what you've been doing.

Reply:
Okie dokie, let's go no line then. Always at the top or at the top of the section that the normal styling would be in e.g. responsive top/bottom styles would go just above the position attribute/display depending